config/initializers/rollbar.rb

```diff
@@ -2,13 +2,14 @@
 Rollbar.configure do |config|
   # Without configuration, Rollbar is enabled in all environments.
   # To disable in specific environments, set config.enabled=false.
-
-  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+  
+  config.access_token = '78e81418fc3344208e2ee10c14354d42'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.